### PR TITLE
BEP 18 suggestions

### PR DESCRIPTION
**Pull request by effigies**
When this is ready to merge, please squash or rebase, so we don't bother polluting the changelog.

Suggestions:

1) Replace bullets with sentences.
2) Language updates for consistency with common-derivatives.
3) Typo fixes (e.g., Connectom -> Connectome)
4) Make examples more specific and conformant to required types

### CODE REVIEW

**Comment by effigies** [comment]
@claramoreau9 I'm unable to request a review from you, but assuming you're the extension developer, I would appreciate your eyes.

**Comment by effigies** [comment]
Cyril, I'm not sure what you mean about removing multiple entries. I don't think I changed anything in that respect. Could you show how you would like it to read?

**Comment by CPernet** [comment]
it's me who read to quickly, you only changed to have to the next line ...  sorry
anyway in the UK bb example (not finished, I know sorry) we wote:  
  
"AnalyticalApproach": ["Whole Genome Sequencing","SNP/CNV Genotypes"],
  
I think it's better, showing multiple options -- is that correct to use the square brackets and list items?
Maybe we should do that here too.
thx

File: src/04-modality-specific-files/08-genetic-descriptor.md [hunk 1]
```diff
@@ -49,33 +56,49 @@ in the `participants.tsv` file by adding optional columns.
...
+| BrainLocation      | OPTIONAL Refers to the location in space of the TissueOrigin                | `MNI coordinate` or a `label` taken from the [Allen Brain Atlas][allen] possibly `layer` to refer to layer-specific gene expression, which can also tie up with laminar fMRI |
+| CellType           | OPTIONAL Describes the type of cell analyzed                                | Value should come from the [cell ontology][ontology]                                                                                                                         |
 
 `genetic_info.json` example:
 
 ```JSON
 {
   "GeneticLevel": "Genetic",
-  "AnalyticalApproach": "SNP Genotypes", "SampleOrigin": "brain",
+  "AnalyticalApproach": "SNP Genotypes",
```
Comment (effigies) [inline src/04-modality-specific-files/08-genetic-descriptor.md]:
```suggestion
  "AnalyticalApproach": ["Whole Genome Sequencing", "SNP/CNV Genotypes"],
```

File: src/04-modality-specific-files/08-genetic-descriptor.md [hunk 2]
```diff
@@ -49,33 +56,49 @@ in the `participants.tsv` file by adding optional columns.
...
+fields:
+
+| Field name         | Definition                                                                  | Values                                                                                                                                                                       |
+| :----------------- | :-------------------------------------------------------------------------- | :--------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| GeneticLevel       | MANDATORY Describes the level of analysis                                   | `Genetic`, `Genomic`, `Epigenomic`, `Transcriptomic`, `Metabolomic`, or `Proteomic`                                                                                          |
+| AnalyticalApproach | OPTIONAL Methodology used to analyse the GeneticLevel                       | Value must be taken from [gapsolr][] under /Study/Molecular Data Type, for instance `SNP Genotypes (Array)` or `Methylation (CpG)`                                           |
```
Comment (effigies) [inline src/04-modality-specific-files/08-genetic-descriptor.md]:
```suggestion
| AnalyticalApproach | OPTIONAL Methodology or methodologies used to analyse the GeneticLevel      | String or list of strings. Each Value must be taken from [gapsolr][] under /Study/Molecular Data Type, for instance `SNP Genotypes (Array)` or `Methylation (CpG)`             |
```

**Comment by CPernet** [comment]
are we good on that one now? anything i need to do

**Pull request merged**